wordcloud.py

```diff
@@ -57,41 +55,41 @@ def process_string(text_to_process):
 
 def determine_occurrences(list_data):
```

Comment:
It might be too long now, but I have changed it to "create_dictionary_based_on_word_occurrence()".

Reply:
For good naming it is best to try to use natural language as much as possible to be easier to understand. In Python the convention is to avoid describing the type of the object in the name. So "create_dictionary" or "create_list" don't sound right.

For variables we already know "word_to_occurrences" / "word_to_count" are good names for a dictionary. We could try to use that style for the function name:

```create_word_to_occurrences()```

It seems like that should end in some noun, like "create_word_to_occurrence_dictionary", but that feels wrong. In this case I like to use the word "map" as in **_mapping_** words to their occurrences.

```create_word_to_occurrences_map()```

At that point we don't need "create" and can simplify to

```map_word_to_occurrences()```